### ISSUE TITLE
Visited anomalies should have a different icon

### ISSUE THREAD

**Issue by NeuralDip** [KSPModStewards/SCANsat#290]
does not need a lot of explanations for this, after 17.6 i lost all visited and recognized anomalies. I still have the old gamedata folder. can i retrieve them from there?
Thanks

**Comment by DMagic1** [comment]
I think it will need more of an explanation. 

By lost anomalies, do you mean that the anomaly symbol no longer shows up on the big map when the anomaly overlay is activated? Was any other data lost? What version were you upgrading from? What version of KSP?

**Comment by NeuralDip** [comment]
no, the anomaly symbol shows fine, but as you can see in the image of the other issue that i opened today, they are all question marks, while in kerbin i have already visited half of them, so expected on the map to be shown the description of the visited anomalies.
i did not experience any other data loss,
i made a clean SCANSat installation
KSP Version :  1.3.0.1804

**Comment by DMagic1** [comment]
The question mark is merely the symbol for an anomaly, like the vessel or waypoint symbols. The description of the anomaly can be seen by hovering the mouse over the question mark (the actual anomaly position is at the center of the question mark circle).

**Comment by NeuralDip** [comment]
ok you are right, it does show by hovering. But i think that the previous way of showing it on the map was easier, because now one has to hover over all of the anomalies to make sure he has visited everything, or to plan the next trip.
